.github/workflows/fast-forward.yml

```diff
@@ -0,0 +1,50 @@
+# Fast-forwards the branch specified in BRANCH_NAME
+# to the github.ref/sha that this workflow is run on.
+# Used as part of the release process, to ensure
+# external query writers can always access a branch of github/codeql
+# that is compatible with the latest stable release.
+name: Fast-forward tracking branch for selected CodeQL version
+on:
+  workflow_dispatch:
+
+jobs:
+  fast-forward:
+    name: Fast-forward tracking branch for selected CodeQL version
```

Comment:
```suggestion
name: Fast-forward tracking branch to selected CodeQL version
on:
  workflow_dispatch:

jobs:
  fast-forward:
    name: Fast-forward tracking branch to selected CodeQL version
```

Reply:
You are correct but I'm going to merge to avoid making you reapprove 😆 Will probably tidy this up as we rename the branch.